src/org/jetbrains/java/decompiler/main/decompiler/ConsoleDecompiler.java

```diff
@@ -60,10 +62,119 @@ else if (args.length > x+1) {
     }
     args = params.toArray(new String[params.size()]);
 
+    if (Arrays.stream(args).anyMatch(arg -> arg.equals("-h") || arg.equals("--help") || arg.equals("-help"))) {
```

Comment:
The content of this if body should be moved out to another method (or maybe even another class) because the main method is getting very long